src/audio_core/audio_renderer.h

```diff
@@ -215,7 +215,7 @@ static_assert(sizeof(UpdateDataHeader) == 0x40, "UpdateDataHeader has wrong size
 class AudioRenderer {
 public:
     AudioRenderer(Core::Timing::CoreTiming& core_timing, AudioRendererParameter params,
-                  Kernel::SharedPtr<Kernel::WritableEvent> buffer_event);
+                  Kernel::SharedPtr<Kernel::WritableEvent> buffer_event, const std::size_t instance_number);
```

Comment:
```suggestion
                  Kernel::SharedPtr<Kernel::WritableEvent> buffer_event, std::size_t instance_number);
```

value-wise const only has a meaning in the definition, not the declaration